docs/config.md

```diff
@@ -0,0 +1,331 @@
+
+Let's assume there are two local nodes already running on different servers of your institutional network, and you want to set up federation across both nodes:
+
+- a node named `"Node Archive"` running on your local computer (localhost), on port `8000` and 
+- a node named `"Node Recruitment"` running on a different computer with the local IP `192.168.0.1`, listening on the default http port `80`. 
+
+In your `local_nb_nodes.json` file you would configure this as follows:
+``` {.json title="local_nb_nodes.json"}
+[
+  {
+    "NodeName": "Node Archive",
+    "ApiURL": "http://host.docker.internal:8000",
+  },
+  {
+    "NodeName": "Node Recruitment",
+    "ApiURL": "http://192.168.0.1"
+  }
+]
+```
+
+!!! warning "Do not use `localhost`/`127.0.0.1` in `local_nb_nodes.json`"
+
+    If the local node API(s) you are federating over is running on the same host machine as your federation API (e.g., the URL to access the node API is http://localhost:XXXX), make sure that you replace `localhost` with `host.docker.internal` in the `"ApiURL"` for the node inside `local_nb_nodes.json`.
+    For an example, see the configuration for the node called `"Node Archive"` above.
+
+
+!!! Info "Nodes that do not need to be manually configured"
+    We maintain a list of public Neurobagel nodes 
+    [here](https://github.com/neurobagel/menu/blob/main/node_directory/neurobagel_public_nodes.json).
+    By default every new `f-API` will lookup this list
+    on startup and include it in the list of nodes to
+    federate over.
+    This also means that you do not have to manually
+    configure public nodes, i.e. you **do not have to explicitly add them** to your `local_nb_nodes.json` file.
+
+To add one or more local nodes to the list of nodes known to your `f-API`, simply add more dictionaries to this file.
+
+
+#### `.env`
+
+`.env` holds environment variables needed for the `f-API` deployment.
+
+``` {.bash .annotate title=".env"}
+# Configuration for f-API
+# Define the port that the f-API will run on INSIDE the docker container (default 8000)
+NB_API_PORT=8000
+# Define the port that the f-API will be exposed on to the host computer (and likely the outside network)
+NB_API_PORT_HOST=8080
+# Chose the docker image tag of the f-API (default latest)
+NB_API_TAG=latest
+
+# Configuration for query tool
+# Define the URL of the f-API as it will appear to a user
+NB_API_QUERY_URL=http://206.12.85.19:8080 # (1)!
+# Chose the docker image tag of the query tool (default latest)
+NB_QUERY_TAG=latest
+# Chose the port that the query tool will be exposed on the host and likely the network (default 3000)
+NB_QUERY_PORT_HOST=3000
+```
+
+1.  When a user uses the graphical query tool to query your
+    f-API, these requests will be sent from the user's machine,
+    not from the machine hosting the query tool.
+
+    Make sure you set the `NB_API_QUERY_URL` in your `.env`
+    as it will appear to a user on their own machine 
+    - otherwise the request will fail.
```

Comment:
As nice as this is, it's hard to notice/find on the docs, maybe it's better to put it in an info/warning section in the main body?

Reply:
ok, Alyssa also pointed out that this whole thing is a duplicate from above, so we'll scrap it